src/profile-logic/process-profile.js

```diff
@@ -1126,10 +1205,21 @@ export function processProfile(
     updateChannel: geckoProfile.meta.updateChannel,
   };
 
+  const profilerOverhead: ProfilerOverhead[] = nullableProfilerOverhead.reduce(
```

Comment:
That was my first try but when I do that, `flow` cannot infer the type of `profilerOverhead` correctly and still thinks it's nullable. On the other hand, `reduce` function is much more stable in flow and works without any problem. That's the error I'm getting when I change that line to `filter`:

```js
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/profile-logic/process-profile.js:1217:48

Cannot assign nullableProfilerOverhead.filter(...) to profilerOverhead because inexact null [1] is incompatible with
exact ProfilerOverhead [2] in array element.

 [1] 1117│   const nullableProfilerOverhead: Array<ProfilerOverhead | null> = [
         :
     1214│   //   },
     1215│   //   []
     1216│   // );
 [2] 1217│   const profilerOverhead: ProfilerOverhead[] = nullableProfilerOverhead.filter(overhead => overhead !== null);
     1218│
     1219│   const result = {
     1220│     meta,

```